tests/tasks/enable_epel.yml

```diff
@@ -4,6 +4,7 @@
   when:
     - ansible_distribution in ['RedHat', 'CentOS']
     - ansible_distribution_major_version in ['7', '8']
+    - ansible_facts["pkg_mgr"] != "ansible.posix.rhel_rpm_ostree"
```

Comment:
This seems redundant, because we already defined the condition `when: ansible_facts["pkg_mgr"] != "ansible.posix.rhel_rpm_ostree"` in some other files.

Reply:
Fixed.